android/res/layout/fragment_report.xml

```diff
@@ -0,0 +1,131 @@
+    android:theme="@style/MwmWidget.ToolbarTheme"
+    tools:ignore="UnusedAttribute">
+    
+    <LinearLayout
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:baselineAligned="false"
+      android:gravity="center_vertical"
+      android:orientation="horizontal">
+      
+      <!-- TODO set correct text (editor_report_problem_title) -->
+      <TextView
+        style="@style/MwmTextAppearance.Toolbar"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_weight="1"
+        android:text="Problem"/>
+      
+      <!-- TODO set correct text (editor_report_problem_send_button) -->
+      <TextView
+        android:id="@+id/save"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:background="?selectableItemBackgroundBorderless"
+        android:padding="@dimen/margin_base"
+        android:text="@string/save"
+        android:textAppearance="@style/MwmTextAppearance.Body1"/>
+    
+    </LinearLayout>
+  
+  </android.support.v7.widget.Toolbar>
+  
+  <ScrollView
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="?panel">
+    
+    <LinearLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:orientation="vertical">
+
+      <LinearLayout
+        android:id="@+id/ll__problems"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        tools:visibility="visible">
+
+        <!-- TODO @string/editor_report_problem_does_not_exist_title -->
+        <TextView
+          android:id="@+id/problem_not_exist"
+          style="@style/MwmWidget.Editor.Problem"
+          android:text="Place does not exist"/>
+
+        <!-- TODO ? -->
+        <TextView
+          android:id="@+id/problem_closed_repair"
+          style="@style/MwmWidget.Editor.Problem"
+          android:text="Closed for repairs"/>
+
+        <!-- TODO ? -->
+        <TextView
+          android:id="@+id/problem_duplicated_place"
+          style="@style/MwmWidget.Editor.Problem"
+          android:text="Duplicating place"/>
+
+        <!-- TODO @string/editor_report_problem_key_other -->
+        <TextView
+          android:id="@+id/problem_other"
+          style="@style/MwmWidget.Editor.Problem"
+          android:text="Other"/>
+      </LinearLayout>
+
+      <LinearLayout
+        android:id="@+id/ll__other_problem"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:visibility="gone">
+
+        <!-- TODO ? -->
+        <EditText
+          android:id="@+id/input"
+          android:layout_width="match_parent"
+          android:layout_height="128dp"
+          android:background="@android:color/white"
```

Comment:
Неа.
> On Mar 15, 2016, at 8:39 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/res/layout/fragment_report.xml <https://github.com/mapsme/omim/pull/2250#discussion_r56206868>:
> 
> > +          android:text="Other"/>
> > +      </LinearLayout>
> > +
> > +      <LinearLayout
> > +        android:id="@+id/ll__other_problem"
> > +        android:layout_width="match_parent"
> > +        android:layout_height="wrap_content"
> > +        android:orientation="vertical"
> > +        android:visibility="gone">
> > +
> > +        <!-- TODO ? -->
> > +        <EditText
> > +          android:id="@+id/input"
> > +          android:layout_width="match_parent"
> > +          android:layout_height="128dp"
> > +          android:background="@android:color/white"
> Не прозрачный ли?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2250/files#r56206868>